_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Log analytics
+nav_order: 12
+---
+
+# Introduction
+
+Data Prepper is an extendable, configurable, and scalable solution for log ingestion into OpenSearch and Amazon OpenSearch Service. Data Prepper supports receiving logs from [FluentBit](https://fluentbit.io/) via the [Http Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md), and processing those logs with a [Grok Processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-processor/README.md) before ingesting them into OpenSearch through the [OpenSearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md).
+
+Here are all of the components for log analytics with FluentBit, Data Prepper, and OpenSearch:
+
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.png)
+
+In the application environment, run FluentBit. FluentBit can be containerized through Kubernetes, Docker, or Amazon ECS. You can also run FluentBit as an agent on EC2. Configure the [FluentBit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper. Then, deploy Data Prepper as an intermediate component and configure it to send the enriched log data to your OpenSearch cluster. From there, use OpenSearch Dashboards to perform more intensive visualization and analysis. 
+
+# Log analytics pipeline 
+
+Log analytic pipelines in Data Prepper are extremely customizable. See the simple pipeline below. 
+
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-ingestion-fluent-bit-data-prepper.png)
+
+# HTTP source
+
+The [HTTP Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md) accepts log data from FluentBit. This source accepts log data in a JSON array format and supports industry-standard encryption in the form of TLS/HTTPS and HTTP basic authentication.
+
+# Processor
+
+Data Prepper 1.2 release and above comes with a [Grok Processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-processor/README.md). The Grok Processor is an invaluable tool to structure and extract important fields from your logs that makes them more queryable.
```

Comment:
The last sentence:
```
The Grok Processor is an invaluable tool to structure and extract important fields from your logs that makes them more queryable.
```

Might read better:
```
The Grok Processor is an invaluable tool used to structure and extract important fields from your logs that makes them more queryable.
```

Reply:
Or maybe "...invaluable tool for structuring and extracting important fields..."